Src/Sarif.PatternMatcher.Sdk/Crc32.cs

```diff
@@ -7,6 +7,10 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
 {
     public static class Crc32
     {
+        public static readonly uint[] Crc32IEEETable = CreateCrcTable(Crc32IEEEPolynomial);
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

Why is this data public?

Reply:
The existing code had it as public but it should have been private as that field is not referenced outside of this class and clearly should be private.
Changed it to private.